ExampleMod/Content/Items/ExampleSignItem.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Terraria.ID;
+using Terraria.ModLoader;
+using Terraria;
+
+namespace ExampleMod.Content.Items
+{
+	public class ExampleSignItem : ModItem
+	{
+		public override void SetStaticDefaults()
+		{
+			Item.DefaultToPlaceableTile(ModContent.TileType<ExampleSign>(), 0);
+		}
+
+		public override void SetDefaults()
+		{
+			Item.width = 26;
+			Item.height = 22;
+			Item.maxStack = 99;
+			Item.useTurn = true;
+			Item.autoReuse = true;
+			Item.useAnimation = 15;
+			Item.useTime = 10;
+			Item.useStyle = ItemUseStyleID.Swing;
+			Item.consumable = true;
+			Item.value = 500;
+			Item.createTile = ModContent.TileType<ExampleSign>();
+		}
```

Comment:
The `DefaultToPlaceableTile` call is misplaced. Move it to `SetDefaults`, and then you can get rid of everything there except `Item.value`. See other tiles in ExampleMod for reference.

Reply:
Updated with https://github.com/tModLoader/tModLoader/pull/4269/commits/b3429f2eb320a9b6257ae914a3d3010ec38a88a3